docs/build/guides/basics/automate-reset-data.mdx

```diff
@@ -0,0 +1,460 @@
+      account,
+      {
+        fee: StellarSdk.BASE_FEE,
+        networkPassphrase: networkPassphrase,
+      },
+    )
+      .addOperation(
+        StellarSdk.Operation.createCustomContract({
+          address: deployerAddress,
+          wasmHash: wasmHash,
+          salt: saltBuffer,
+        }),
+      )
+      .setTimeout(30)
+      .build();
+
+    const createContractTx = await server.prepareTransaction(
+      createContractTransaction,
+    );
+    createContractTx.sign(deployer);
+    let createContractResponse = await server.sendTransaction(createContractTx);
+    const returnContractResponse = await getTxsReports(
+      createContractResponse,
+      `Contract Deployed...`,
+      server,
+    );
+    const contractBuffer = returnContractResponse.returnValue._value._value;
+    const contractId = StellarSdk.StrKey.encodeContract(contractBuffer);
+    const contract = new StellarSdk.Contract(contractId);
+
+    // Invoke Contract
+    const invokeContractTransaction = new StellarSdk.TransactionBuilder(
+      account,
+      {
+        fee: StellarSdk.BASE_FEE,
+        networkPassphrase: networkPassphrase,
+      },
+    )
+      .addOperation(
+        contract.call("hello", StellarSdk.xdr.ScVal.scvSymbol("World")),
+      )
+      .setTimeout(30)
+      .build();
+
+    const invokeContractTx = await server.prepareTransaction(
+      invokeContractTransaction,
+    );
+    invokeContractTx.sign(deployer);
+    let invokeContractResponse = await server.sendTransaction(invokeContractTx);
+    const returnInvokeContractResponse = await getTxsReports(
+      invokeContractResponse,
+      `Invoke Contract.`,
+      server,
+    );
+
+    const functionValues = returnInvokeContractResponse.returnValue._value
+      .map((item) => item._value)
+      .filter(Boolean);
+
+    return {
+      contractId: contractId,
+      returnValues: functionValues,
+    };
+  } catch (error) {
+    console.error("Error in contract deployment and invocation:", error);
+    throw error;
+  }
+}
+
+async function automateSetup() {
+  try {
+    // const networkRPC = "USE EITHER FUTERNET OR TESTNET RPC"
+    const networkRPC = "https://soroban-testnet.stellar.org";
+    // Example
+    // FOR FUTERENET - https://rpc-futurenet.stellar.org
+    // FOR TESTNET - https://soroban-testnet.stellar.org
+
+    //Initiate Server
+
+    const server = new StellarSdk.SorobanRpc.Server(networkRPC);
+
+    //Check Network Status
+    console.log("Checking network health...");
+    const health = await server.getHealth();
+    console.log("Network health:", health);
+
+    // const networkURL = "USE EITHER FUTERNET OR TESTNET URL"
+    const networkURL = "https://friendbot.stellar.org";
+    // Example
+    // FOR FUTERENET - https://friendbot-futurenet.stellar.org
+    // FOR TESTNET - https://friendbot.stellar.org
+
+    const accountOne = await createAccount(networkURL);
+    const accountTwo = await createAccount(networkURL);
```

Comment:
I mentioned in my previous review, that (at least) `accountOne` **needs** to not be created by `Keypair.random()`. We're trying to make this a useful script for people who need to setup predictable things after a network reset. Among the most important things that should be "predictable" is an asset's issuer address.